### ISSUE TITLE
Generating Story with {#each} block

### ISSUE THREAD

**Issue by YunnXairou** [storybookjs/addon-svelte-csf#46]
Hello,
I am planning to make a component with some data example. I saw the plugin [storybook-fixtures](https://storybook.js.org/addons/storybook-fixtures) but I did not manage to make it work for svelte. I am planning to simulate it with {#each} block but it too does not work.

Is it really impossible or did I do something wrong ? 
If it's not possible, could we add a feature like this ?

Here are the source if you want to test:

# Calendar.stories.svelte
```svelte
<script lang="ts">
	import { Meta, Template, Story } from '@storybook/addon-svelte-csf';

	import Calendar from './Calendar.svelte';
	import calendarData from './fixtures.json';
</script>

<Meta title="Calendar" />

<Template let:args>
	<Calendar {...args} />
</Template>

{#each Object.entries(calendarData) as [name, args]}
	<Story {name} {args} />
{/each}
```

# Calendar.svelte
```svelte
<script lang="ts">
	export let title = '';
</script>

{title}
```

# fixtures.json
```json
{
	"Tiger": {
		"title": "Tiger"
	},
	"Lion": {
		"title": "Lion"
	},
	"Jaguar": {
		"title": "Jaguar"
	},
	"Leopard": {
		"title": "Leopard"
	},
	"Snow Leopard": {
		"title": "Snow leopard"
	}
}
```

**Comment by j3rem1e** [comment]
and the addon-fixture can't be used too because the story context is not exposed in a <Story/> or <Template/>.

**Comment by shilman** [comment]
:rocket: Issue was released in [`v1.1.2`](https://github.com/storybookjs/addon-svelte-csf/releases/tag/v1.1.2) :rocket:
